itests/src/test/java/fi/metatavu/edelphi/test/ui/base/index/ImportantBulletinsTestsBase.java

```diff
@@ -0,0 +1,71 @@
+package fi.metatavu.edelphi.test.ui.base.index;
+
+import java.time.OffsetDateTime;
+import java.util.Date;
+
+import org.junit.Test;
+
+import fi.metatavu.edelphi.test.mock.DelfoiBulletinMocker;
+import fi.metatavu.edelphi.test.ui.base.AbstractUITest;
+
+public class ImportantBulletinsTestsBase extends AbstractUITest {
+  
+  private DelfoiBulletinMocker delfoiBulletinMocker = new DelfoiBulletinMocker();
+  
+  @Test
+  public void testImportantBulletinPopup() {
+    Date created = toDate(2017, 5, 3);
+    
+    delfoiBulletinMocker.createBulletin("Normal bulletin", "<p>Normal bulletin content</p>", 1l, created, Boolean.FALSE, null);
+    
+    // Normal bulletin should not be visible
+    
+    login(ADMIN_EMAIL);
+    navigate("/");
+    assertNotPresent(".important-bulletin-dialog");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ".important-bulletin-dialog" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)